charts/mozmoderator/templates/secret.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.externalSecret.create -}}
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: {{ include "mozmoderator.fullname" . }}
+spec:
+  backendType: secretsManager
+  dataFrom:
+  - /{{ .Values.environment }}/moderator
+{{- else }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "mozmoderator.fullname" . }}
+data:
```

Comment:
@The-smooth-operator I was surprised to see any values here, if ExternalSecret is pulling from secretsmanager, I suppose maybe these are default vaules? can you describe what this is a bit?

Reply:
Yes. The idea here is to allow both using secretsmanager or a Kubernetes secret. Here I'm stating that or secretsmanager is used, or otherwise we roll this default secret.

The default secret is used by the end to end test. When the application starts, it opens a connection to the database and needs the Database env var. There others are not really used, but the application is expecting the environment variables to exist. 

The connection string values are matching the ones defined in the MySQL bootstrap values [here](https://github.com/mozilla-it/helm-charts/pull/39/commits/1a0fbfed5810a6d6875ca0172adac5065ee03b74#diff-e52d1c600bb15e699ae0d33fae8b7b53R56-R60)